### PR TITLE
Update generate-training-data.py

### PR DESCRIPTION
**Pull request by vijay1827**
same image is saved after running the loop

### CODE REVIEW

**Comment by vijay1827** [comment]
PARAS and VIJAY

File: preprocess-and-train.py [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
 # or that you write your code in such a way that you pre-process on the fly
 # pre-processing on the fly is possible because this is small project and
 # not a lot of images will be there as well.
 def preprocess(img):
-    # your code goes here
+    width = 225
+    height = 225
+    dimensions = (width, height)
+    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+    img = cv2.resize(img, dimensions, interpolation=cv2.INTER_CUBIC)
```
Comment (mukeshmk) [inline preprocess-and-train.py:17]: what other preprocessing techniques did you try?
and what were the results of those techniques? @vijay1827

Comment (vijay1827) [reply preprocess-and-train.py:17]: Using cv2.COLOR_BGR2HSV color space accuracy is 97.75
AND using cv2.COLOR_BGR2GRAY color space accuracy is 96.7

Comment (vijay1827) [reply preprocess-and-train.py:17]: using cv2.COLOR_BGR2RGB color space accuracy is 92.00

Comment (vijay1827) [reply preprocess-and-train.py:17]: using GaussianBlur accuracy 98

**Comment by mukeshmk** [comment]
Review Accepted: Closing the PR as this is just for review purpose.

**Pull request closed without merging**